src/Console/MacrosCommand.php

```diff
@@ -187,8 +194,12 @@ protected function generateFunction($name, $parameters, $type = '', $returnType
         }
 
         $this->write(")");
-        if ($returnType) {
-            $this->write(": $returnType");
+        if (version_compare(PHP_VERSION, '7.1', '<')) {
+            if ($returnType) {
+                $this->write(": \\$returnType");
+            }
+        } elseif ($returnType instanceof \ReflectionNamedType){
+            $this->write(": \\{$returnType->getName()}");
```

Comment:
In other parts of the code, method calls are never inline but appended to the string instead - just for the sake of consistency I think it'd be best to stick to that format.